packages/components/molecules/f-card-with-content/README.md

```diff
@@ -69,15 +73,31 @@ There may be props that allow you to customise its functionality.
 
 The props that can be defined are as follows (if any):
 
-| Prop  | Type  | Default | Description |
-| ----- | ----- | ------- | ----------- |
+| Prop                  | Type     | Default | Description                                                   |
+| --------------------- | -------- | ------- | ------------------------------------------------------------- |
+| `cardHeading`         | `String` | `''`    | If given, will render an `h1` tag within the card.            |
```

Comment:
Might not be needed for Contact Prefs, but may be good to have the heading type definable (so letting peoples choose `h2`/`h3` for instance would likely be useful. Could be a separate ticket potentially to do this – whatever you think makes the most sense.

Reply:
Yep, I considered it but I think it makes sense to add this when it's required.

Reply:
Can you make a separate ticket for it in EE?

Reply:
Ticket made 👍🏻